{"name": "Вход с невалидными данными", "status": "passed", "description": "Проверяю возникновение ошибки", "steps": [{"name": "Захожу в учетную запись с данными fsfuwqrcth, esumqvkhkg", "status": "passed", "start": 1730196703335, "stop": 1730196703453}], "start": 1730196703335, "stop": 1730196703453, "uuid": "d4b722e0-92d3-4020-b3d8-6939a3fb7afc", "historyId": "97611c5a78a69c65b918f60c7e624a02", "testCaseId": "97611c5a78a69c65b918f60c7e624a02", "fullName": "tests.couriers.test_login_couriers.TestCouriersLogin#test_wrong_login", "labels": [{"name": "severity", "value": "normal"}, {"name": "suite", "value": "Курьер"}, {"name": "subSuite", "value": "Логин курьера"}, {"name": "parentSuite", "value": "tests.couriers"}, {"name": "host", "value": "Konstantin"}, {"name": "thread", "value": "15044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_login_couriers"}]}